src/seedu/addressbook/commands/Command.java

```diff
@@ -14,13 +17,17 @@
 public class Command {
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
-    private int targetIndex = -1;
+//    private int[] targetIndexes;
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.